src/frontends/tensorflow/tests/CMakeLists.txt

```diff
@@ -22,10 +22,11 @@ ov_add_test_target(
 )
 
 # Test model generating
-ov_check_pip_package(REQUIREMENT tensorflow
-                     MESSAGE_MODE WARNING
-                     WARNING_MESSAGE "TensorFlow frontend unit tests will be skipped"
-                     RESULT_VAR tensorflow_FOUND)
+
+ov_check_pip_packages(REQUIREMENTS_FILE "${CMAKE_CURRENT_SOURCE_DIR}/requirements.txt"
+                      MESSAGE_MODE WARNING
+                      WARNING_MESSAGE "TensorFlow frontend unit tests will be skipped"
+                      RESULT_VAR paddlepaddle_FOUND)
```

Comment:
why do we need this change?

Reply:
we need to check all the requirements from TF's requirements.txt and with exact versions

Reply:
What will be if these requirements are not installed? Will these unti-tests be skipped. Can we fail in this case to avoid such cases when the testing is missed?

Reply:
When models are not converted because of framework absence, best we can do is build binaries at least and inform users that tests will fail